Meadowlark-js/packages/meadowlark-core/src/handler/Query.ts

```diff
@@ -14,6 +14,7 @@ import { FrontendRequest } from './FrontendRequest';
 import { FrontendResponse } from './FrontendResponse';
 
 const moduleName = 'Query';
+const TOTAL_COUNT_HEADER_NAME: string = 'total-count';
```

Comment:
Added - so that it matched most existing headers. totalCount would become 'totalcount' in the headers, which looks weird to me.